mappings/net/minecraft/server/dedicated/ServerPropertiesHandler.mapping

```diff
@@ -49,8 +49,6 @@ CLASS net/minecraft/class_3806 net/minecraft/server/dedicated/ServerPropertiesHa
 	FIELD field_26899 textFilteringConfig Ljava/lang/String;
 	FIELD field_27845 requireResourcePack Z
 	FIELD field_33432 resourcePackPrompt Ljava/lang/String;
-	METHOD <init> (Ljava/util/Properties;)V
-		ARG 1 properties
```

Comment:
Is this just a byproduct of dropping invalid mappings?

Reply:
Yep it is

Reply:
If this is, you can keep it. If other prs have the same change, it's acceptable as git can handle such merges properly.